### PR TITLE
Close #10

### LINKED ISSUE #10: Convert pvdataset archives
The Bruker -> DICOM conversion currently starts from a directory. It would be helpful if it could also input a `pvdataset` archive.

### PR DESCRIPTION
**Pull request by farid67**

### CODE REVIEW

File: src/cli/bruker2dicom [hunk 1]
```diff
@@ -58,13 +61,34 @@ def main():
     logging.getLogger().setLevel(verbosity.upper())
     
     function = arguments.pop("function")
+    source = arguments.pop("source")
+    
+    #create tmp directory in the current location
+    extractdir = tempfile.mkdtemp("","",".")
```
Comment (lamyj) [inline src/cli/bruker2dicom]: Creating a temporary directory in the current working directory (`"."`) is usually a bad idea: we have no way to ensure that this is writable. Also, it is not necessary to remove the `prefix` argument. The line should read:
```python
extractdir = tempfile.mkdtemp()
```

File: src/cli/bruker2dicom [hunk 2]
```diff
@@ -58,13 +61,34 @@ def main():
...
-        if verbosity == "debug":
-            raise
-        else:
-            parser.error(e)
+        # try to extract the archive
+        zip_obj = zipfile.ZipFile(source)
+        zip_obj.extractall(extractdir)
+    except:
+        # source is not a pvDataSet archive, so it should be a "normal" directory
+            arguments['source'] = source
```
Comment (lamyj) [inline src/cli/bruker2dicom]: The indentation looks weird: is there a mix of tabs and spaces, or just extra spaces?

File: src/cli/bruker2dicom (hunk 1 above)
Comment (farid67) [inline src/cli/bruker2dicom]: Ok but letting it like that will create the directory under /tmp so it can cause problems for Windows users?

Comment (lamyj) [inline src/cli/bruker2dicom]: No, the root directory is system-dependent: it's not mentioned in the `mkdtemp` doc, but it's explicit in the [mkstemp doc](https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp).

**Pull request merged**